docs/Tags.md

```diff
@@ -1,4 +1,48 @@
+[DE](DE%3ATags.md)
+
 # Tags and Tag-groups
 
-***
-Switch Language: [DE](DE%3ATags.md)
\ No newline at end of file
+## Creating time bookings with tags
+
+Time bookings can be created with one or multiple `tags`. Tags are intended to help identify a time booking and also associate multiple time bookings with each other.
+
+When recording a time booking, either manual tags can be created by entering text in the corresponding input field, or a tag assigned to the selected project can be chosen.
+
+![Creating tags](images/Lasius_TimeBooking_Create_Tags.png)
+
+The list of available tags can also be opened using the down arrow key or the space bar.
+
+*If the same text is entered as a tag when manually recording, it will be identified as the same tag.*
+
+By configuring an [integration](Integrations.md), tags can also be automatically imported from a third-party application like Gitlab, Plane, or Jira, making them available for selection. For externally defined tags, a link is also stored behind the tag, allowing reference to the corresponding application when clicked.
+
+## Benefits of tags
+
+The tags assigned to time bookings are visible in the current time bookings and past time bookings in the list.
+
+![Tags in time bookings](images/Lasius_TimeBooking_List_Bookings.png)
+![Tags in time bookings](images/Lasius_TimeBooking_List_Tags.png)
+
+On the other hand, tags are also used to automatically group time bookings in the [statistics](Statistics.md#tags), providing a quick overview of time bookings.
+
+
+## Tag groups
+
+A tag group is, as the name suggests, a grouping of multiple `tags` that are displayed together under a single name. In the selection list of available tags and in the list of time bookings, tag groups are only displayed under the group name.
+
+However, the corresponding assigned time booking is automatically assigned to all tags assigned to the tag group, so that they are taken into account in the [statistics](Statistics.md#tags) for all tags.
```

Comment:
3 times "assigned" in the same sentence made my eyes bleed. "_Can we remove at least the first one: However, the corresponding time booking is automatically assigned to all tags assigned to the tag group, so they are included in the statistics._"